NGitLab.Mock/MergeRequestChangeCollection.cs

```diff
@@ -10,11 +11,25 @@ public MergeRequestChangeCollection(GitLabObject parent)
         }
 
         public Change Add(string diff)
+        {
+            return Add(diff, oldPath: null, newPath: null, changeKind: null);
+        }
+
+        public Change Add(string diff, string oldPath, string newPath, ChangeKind? changeKind)
         {
             if (diff is null)
                 throw new ArgumentNullException(nameof(diff));
 
-            var change = new Change() { Diff = diff };
+            var change = new Change
+            {
+                Diff = diff,
+                NewPath = newPath,
+                OldPath = oldPath,
+                DeletedFile = changeKind.HasValue && changeKind.Value == ChangeKind.Deleted,
```

Comment:
You can simplify the condition as follows (same for other 2)
```suggestion
                DeletedFile = changeKind.Value == ChangeKind.Deleted,
```

Reply:
changeKind is nullable, wouldn't it trigger an exception?

Reply:
Nope, `T? item1 == T item2` is equivalent to `item1.HasValue && item1.Value == item2`

Reply:
Oh, I think it's just the suggestion given by louis:
`changeKind.Value == ChangeKind.Deleted` => `changeKind== ChangeKind.Deleted`

Will update that.